tools/binlog/tidb-binlog-local.md

```diff
@@ -0,0 +1,287 @@
+---
+title: TiDB-Binlog Local 部署方案
+category: tools
+---
+
+# TiDB-Binlog Local 部署方案
+
+## TiDB-Binlog-local 简介
+
+TiDB-Binlog 用于收集 TiDB 的 Binlog，并提供实时备份和同步功能的商业工具。
```

Comment:
ditto?

Reply:
Fixed in https://github.com/pingcap/docs-cn/pull/1264/commits/71160641b4a7643e3b78cfeb894849ef08a63879